app/assets/scripts/views/home.js

```diff
@@ -53,7 +53,9 @@ module.exports = React.createClass({
         'provider': Joi.string().required().label('Provider'),
         'contact-type': Joi.string().required().valid('uploader', 'other'),
         'contact-name': Joi.label('Name').when('contact-type', { is: 'other', then: Joi.string().required() }),
-        'contact-email': Joi.label('Email').when('contact-type', { is: 'other', then: Joi.string().email().required() })
+        'contact-email': Joi.label('Email').when('contact-type', { is: 'other', then: Joi.string().email().required() }),
+        'license': Joi.string().required().label('License'),
+        'tags': Joi.string().required().label('Tags')
```

Comment:
@nbumbarger Are tags supposed to be required? On the uploader-api they are not https://github.com/hotosm/oam-uploader-api/pull/57/files#diff-816462b1832fd8fadda1d5d0d6086592R18